src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx

```diff
@@ -134,8 +126,8 @@ function StatusClearAfterPage({currentUserPersonalDetails, customStatus}) {
             if (mode.value === CONST.CUSTOM_STATUS_TYPES.CUSTOM) {
                 User.updateDraftCustomStatus({clearAfter: DateUtils.getOneHourFromNow()});
             } else {
-                const selectedRange = _.find(statusType, (item) => item.value === mode.value);
-                const calculatedDraftDate = DateUtils.getDateFromStatusType(selectedRange.value);
+                const selectedRange = statusType.find((item) => item.value === mode.value);
+                const calculatedDraftDate = DateUtils.getDateFromStatusType(selectedRange?.value ?? 'never');
```

Comment:
Why `'never'`?

Reply:
I thought that if there is no `selectedRange` we shouldn't clear the status

Reply:
Let's use the constant then! `CONST.CUSTOM_STATUS_TYPES.NEVER`